custom/apps/crs_reports/reports.py

```diff
@@ -189,9 +190,7 @@ def base_filters(self):
         return filters
 
     def date_to_json(self, date):
-        return tz_utils.adjust_datetime_to_timezone\
-            (date, pytz.utc.zone, self.timezone.zone).strftime\
-            ('%d-%m-%Y') if date else ""
+        return ServerTime(date).user_time(self.timezone).done().strftime('%d-%m-%Y') if date else ""
```

Comment:
I think this should be the same as here: https://github.com/dimagi/commcare-hq/pull/6035/files#diff-64271a8d193a39ddd8979432c1da0d21R323

The date is a date in the case e.g. 'date_birth', 'opened_on', 'modified_on', 'closed_on'